Releasenotes v8.13 and minor bugfix RN v8.9 (#826)
MichaIng/DietPi-Docs: docs/releases/v8_13.md

- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**Docker**](../../software/programming/#docker) :octicons-arrow-right-16: Resolved an issue where the install failed if the kernel did not support nftables. Since iptables is installed as dependency along with the Docker package, it cannot be configured first to use the legacy API where needed. The Docker service is now masked before the package is installed, to prevent it from starting during the package install, before we can configure iptables. Many thanks to @tggjifdhn for reporting this issue: <https://github.com/MichaIng/DietPi/issues/6013>
- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**myMPD**](../../software/media/#mympd) :octicons-arrow-right-16: Resolved an issue where APT updates failed since the myMPD APT repository key expired on 2023-01-01. Many thanks to @cpsjones for reporting this issue: <https://github.com/MichaIng/DietPi/issues/6044>

> - [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**Homer**](../../software/system_stats/#homer) :octicons-arrow-right-16: Resolved an issue where reinstalls failed. Many thanks to @JappeHallunken for reporting this issue: <https://github.com/MichaIng/DietPi/issues/6060>

Reviewer: ```suggestion
- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**Homer**](../../software/system_stats/#homer) :octicons-arrow-right-16: Resolved an issue where reinstalls failed. Many thanks to @JappeHallunken for reporting this issue: <https://github.com/MichaIng/DietPi/issues/6060>
- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**Grafana**](../../software/hardware_projects/#grafana) :octicons-arrow-right-16: Resolved an issue where APT updates failed if Grafana was installed, since the repo key was rotated on January 12: <https://grafana.com/blog/2023/01/12/grafana-labs-update-regarding-circleci-security-updates/>
```

- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**Docker**](../../software/programming/#docker) :octicons-arrow-right-16: Resolved an issue where the install failed if the kernel did not support nftables. Since iptables is installed as dependency along with the Docker package, it cannot be configured first to use the legacy API where needed. The Docker service is now masked before the package is installed, to prevent it from starting during the package install, before we can configure iptables. Many thanks to @tggjifdhn for reporting this issue: <https://github.com/MichaIng/DietPi/issues/6013>
- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**myMPD**](../../software/media/#mympd) :octicons-arrow-right-16: Resolved an issue where APT updates failed since the myMPD APT repository key expired on 2023-01-01. Many thanks to @cpsjones for reporting this issue: <https://github.com/MichaIng/DietPi/issues/6044>

> As always, many smaller code performance and stability improvements, visual and spelling fixes have been done, too much to list all of them here. Check out all code changes of this release on GitHub: <https://github.com/MichaIng/DietPi/pull/5985>

Reviewer: Release PR has been created: https://github.com/MichaIng/DietPi/pull/6053
```suggestion
As always, many smaller code performance and stability improvements, visual and spelling fixes have been done, too much to list all of them here. Check out all code changes of this release on GitHub: <https://github.com/MichaIng/DietPi/pull/6053>
```

> 
> 
> xxx

Reviewer: Resolve Markdown lint error:
```suggestion
```

> As always, many smaller code performance and stability improvements, visual and spelling fixes have been done, too much to list all of them here. Check out all code changes of this release on GitHub: <https://github.com/MichaIng/DietPi/pull/5985>
> 
> 
> xxx

Reviewer: Release PR has been created: https://github.com/MichaIng/DietPi/pull/6053
```suggestion
As always, many smaller code performance and stability improvements, visual and spelling fixes have been done, too much to list all of them here. Check out all code changes of this release on GitHub: <https://github.com/MichaIng/DietPi/pull/6053>
```
